components/ui/EmbedPreview.tsx

```
@@ -0,0 +1,186 @@
+            type="button"
+            aria-label="Close full-page embed"
+            onClick={() => setShowFullPage(false)}
+            className="text-[#E8DCC4] hover:text-[#C4B8A0] transition-colors"
+          >
+            <svg
+              className="w-5 h-5"
+              fill="none"
+              stroke="currentColor"
+              viewBox="0 0 24 24"
+            >
+              <path
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)